release/README.md

```diff
@@ -0,0 +1,12 @@
+# Release Process
```

Comment:
https://github.com/kubeflow/kfserving/blob/master/test/scripts/build-kfserving.sh is triggered  to push kfserving image to gcr.io by each PR submit and merged event.
Do you plan to drop it?

Reply:
Oh, I thought we just test the build, did not know it also publishes to registry.

Reply:
Oh I completely missed this. My feeling is that the current test scripts are far more complicated than they need to be. Maybe this is simply due to my own lack of deep understanding of what's going on. How do you feel about the proposed change Hougang? If you don't think it's any simpler and think diverging from Kubeflow's ksonnet based approach is a bad idea, I'm happy to not submit this.

Reply:
For me, it works well now. I don't think we should  spend too much effort for another such a counterpart. If you really don't like it, maybe we can discuss in Kubeflow meeting about it to refactor kubeflow CI infra :)
Thanks

Reply:
Agreed I want to avoid too much investment, but I also want to avoid dependencies on overly complex solutions. I'll bring it up in an upcoming kubeflow meeting. Given how simple this is, do you mind having both for the short term? I'm curious to see how well Cloud Build Triggers perform and see if there are any hiccups.

Reply:
sure, it's OK for me.